src/main/java/mate/academy/bookshop/model/ShoppingCart.java

```diff
@@ -0,0 +1,44 @@
+package mate.academy.bookshop.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.NamedAttributeNode;
+import jakarta.persistence.NamedEntityGraph;
+import jakarta.persistence.NamedSubgraph;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.Set;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "shopping_carts")
+@NamedEntityGraph(
+        name = "shoppingCartWithItemsAndBooks",
+        attributeNodes = {
+                @NamedAttributeNode(value = "user"),
+                @NamedAttributeNode(value = "cartItems", subgraph = "cartItemsGraph")
+        },
+        subgraphs = {
+                @NamedSubgraph(
+                        name = "cartItemsGraph",
+                        attributeNodes = @NamedAttributeNode(value = "book")
+                )
+        }
+)
```

Comment:
Looks like it is kind of overcomplicating, it is really necessary to add this annotation?

Reply:
Yes, without this entitygraph setup i will get LazyInitializationException